airflow/providers/google/suite/hooks/drive.py

```diff
@@ -197,7 +197,7 @@ def upload_file(self, local_location: str, remote_location: str) -> str:
             parent = "root"
 
         file_metadata = {"name": file_name, "parents": [parent]}
-        media = MediaFileUpload(local_location)
+        media = MediaFileUpload(local_location, resumable=True)
```

Comment:
Any drawbacks for using resumable? Should it be turned into a parameter maybe?

Reply:
afaik it has no drawbacks. But we can use a parameter, of course.

Reply:
Quick look and at least one issue https://github.com/googleapis/google-api-python-client/issues/625 which has been closed by inactivity. I think it **may** change behaviour, so having a parameter (defaulting to False) is much better.

Also that means some unit test shoudl be added.

Reply:
Also - from that thread - it sems advisable that we could override the chunk_size